src/components/map/layers/new/NewRoutePathLayer.tsx

```diff
@@ -20,8 +20,8 @@ interface IRoutePathLayerProps {
 @observer
 export default class RoutePathLayer extends Component<IRoutePathLayerProps> {
     private renderRoutePathLinks(
-        routePathLinks: IRoutePathLink[]|null,
-        { isNeighbor }: { isNeighbor: boolean }) {
+        { isNeighbor }: { isNeighbor: boolean },
```

Comment:
Why is this an object?

Reply:
If we want to explicitly explain what something is, rather do `const isNeighbor = true` when calling this

Reply:
Ok, I'm finding references to doing this.. But they usually just take one object with everything instead of an object with one value.

Reply:
Fixed, removed that ugly isNeighbor boolean completely.